libraries/AP_InertialSensor/AP_InertialSensor.h

```diff
@@ -211,6 +214,15 @@ class AP_InertialSensor : AP_AccelCal_Client
     // get the accel filter rate in Hz
     uint16_t get_accel_filter_hz(void) const { return _accel_filter_cutoff; }
 
+    // harmonic notch current center frequency from IMU 1
+    float get_gyro_harmonic_notch_center_freq_hz(void) const { return _calculated_harmonic_notch_freq_hz; }
+
+    // harmonic notch reference center frequency
+    float get_gyro_harmonic_notch_reference_freq_hz(void) const { return _harmonic_notch_filter.center_freq_hz(); }
```

Comment:
Hmmn, ok. Will have to think about this.

Reply:
The harmonic notch itself doesn't really know anything about the reference vs center frequency - it only has one. It is the IMU code that is making a distinction. So I'm comfortable with the naming but could change them to center_freq_hz and current_center_freq_hz if you think that would be better?

Reply:
Maybe this should be:

    float get_gyro_dynamic_notch_center_freq_hz(void) const { return _calculated_harmonic_notch_freq_hz; }

     // harmonic notch reference center frequency
    float get_gyro_harmonic_notch_center_freq_hz(void) const { return _harmonic_notch_filter.center_freq_hz(); }

We only need the dynamic call because we are using it dynamically otherwise we would just use the second.

Reply:
Ok done